pydantic/_internal/_core_utils.py

```diff
@@ -280,12 +280,37 @@ def handle_dict_schema(self, schema: core_schema.DictSchema, f: Walk) -> core_sc
             schema['values_schema'] = self.walk(values_schema, f)
         return schema
 
-    def handle_function_schema(self, schema: AnyFunctionSchema, f: Walk) -> core_schema.CoreSchema:
```

Comment:
This method was in reality never used: there's no `'function'` core schema. Instead, a method for each type was created.